src/nvim/tui/input.c

```diff
@@ -684,6 +687,29 @@ HandleState handle_background_color(TermInput *input)
   return kComplete;
 }
 
+static void handle_osc_event(TermInput *input, const TermKeyKey *key)
+{
+  assert(input);
+
+  const char *str = NULL;
+  if (termkey_interpret_string(input->tk, key, &str) == TERMKEY_RES_KEY) {
+    assert(str != NULL);
+
+    // Send an event to the TUI. This will update the v:termresponse variable and fire the
```

Comment:
`to nvim core`

Reply:
bcc0f5e754eaf0e98f208b62dcb1965b88f807c2